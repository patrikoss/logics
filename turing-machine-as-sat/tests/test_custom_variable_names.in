4 3 7
has bob a cat
q1 q2 q3
q1
q3
bob q1 cat q2 1
bob q1 cat q2 -1
has q2 has q2 1
has q2 has q1 -1
has q1 has q1 1
a q2 a q2 1
cat q2 bob q3 0
4
bob has a cat

