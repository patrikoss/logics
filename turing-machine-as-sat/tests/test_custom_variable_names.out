YES

bob|q1 has|  a|  cat| 
cat|  has|q2 a|  cat| 
cat|  has|  a|q2 cat| 
cat|  has|  a|  cat|q2
cat|  has|  a|  bob|q3
